content/api/trafiklab-apis/gtfs-sweden/static.md

```diff
@@ -0,0 +1,34 @@
+---
+title: GTFS Sweden Static data 
+aliases: 
+  - "/api/gtfs-sweden-static"
```

Comment:
These were only used as a fallback from the old Trafiklab website, and are not needed on new APIs